packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/RevealedData.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { ClipboardCopy, ClipboardCopyVariant, TextInput } from '@patternfly/react-core';
+
+export const RevealedData: React.FC<{ value: string }> = ({ value }) => {
+  const { t } = useForkliftTranslation();
+
+  return value ? (
+    <ClipboardCopy
+      isReadOnly
+      hoverTip={t('Copy')}
+      clickTip={t('Copied')}
+      isCode
+      variant={value && value.length > 128 ? ClipboardCopyVariant.expansion : undefined}
+    >
+      {value}
+    </ClipboardCopy>
+  ) : (
+    <TextInput value={''} type="text" isDisabled />
```

Comment:
I want to add something to indicate leaving this input empty is intentional and not ane error, can you think of something that will indicate this input is empty because it is missing in the CR and not because of some unexpected error ?